azure-webapp-maven-plugin/README.md

```diff
@@ -1,182 +1,175 @@
+## Quick Start
+1. Make sure you have already logged in. You can use the Azure CLI 2.0 for authentication. More authentication methods can be found [here](../docs/common-configuration.md).
 
+   - Install the Azure CLI 2.0 by following the instructions in the [Install Azure CLI 2.0](https://docs.microsoft.com/cli/azure/install-azure-cli) article.
+   - Run the following commands to log into your Azure subscription：
+    ```
+        $ az login
+        $ az account set --subscription <subscription Id>
+    ```
+2. You could create a new Azure App Service and fill in the information in the `pom.xml`. This configuration can also be generated by [`azure-webapp:config`](#azure-webappconfig)
     ```xml
-    <project>
-       ...
-       <packaging>war</packaging>
-       ...
-       <build>
-           <plugins>
-               <plugin>
-                   <groupId>com.microsoft.azure</groupId>
-                   <artifactId>azure-webapp-maven-plugin</artifactId>
-                   <!-- check Maven Central for the latest version -->
-                   <version>1.5.1</version>
-                   <configuration>
-                       <schemaVersion>v2</schemaVersion>
-                       <resourceGroup>your-resource-group</resourceGroup>
-                       <appName>your-app-name</appName>
-                       <region>westus</region>
-                       <pricingTier>P1V2</pricingTier>
-                       <runtime>
-                           <os>linux</os>
-                           <javaVersion>jre8</javaVersion>
-                           <webContainer>tomcat 8.5</webContainer>
-                       </runtime>
-                       <deployment>
-                           <resources>
-                               <resource>
-                                   <directory>${project.basedir}/target</directory>
-                                   <includes>
-                                       <include>*.war</include>
-                                   </includes>
-                               </resource>
-                           </resources>
-                       </deployment>                        
-                   </configuration>
-               </plugin>
-           </plugins>
-       </build>
-    </project> 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.microsoft.azure</groupId>
+                <artifactId>azure-webapp-maven-plugin</artifactId>
+                <!-- check Maven Central for the latest version -->
+                <version>1.5.2</version>
+                <configuration>
+                    <schemaVersion>v2</schemaVersion>
+                    <resourceGroup>'your-resource-group'</resourceGroup>
+                    <appName>'your-app-name'</appName>
+                    <region>westeurope</region>
+                    <pricingTier>P1V2</pricingTier>
```

Comment:
`pricingTier` defaults to `P1V2`, this line can be removed

Reply:
Here is just show how config the pricingTier, so I think we can reserve it.